packages/bootstrap-vue-next/src/components/BToast/BToast.vue

```diff
@@ -178,18 +175,26 @@ watch(isActive, (newValue) => {
 const computedClasses = computed(() => [
   resolvedBackgroundClasses.value,
   {
-    show: isToastVisible.value,
+    'show': isToastVisible.value,
+    'fade-enter-active': !noFadeBoolean.value,
   },
 ])
 
+const fadeClasses = computed(() => {
+  if (!noFadeBoolean.value) {
+    return 'fade-leave-active'
+  }
+
+  return ''
+})
```

Comment:
```suggestion
const fadeClasses = computed(() => ({
  'fade-leave-active': !noFadeBoolean.value
}))
```
Same question as above